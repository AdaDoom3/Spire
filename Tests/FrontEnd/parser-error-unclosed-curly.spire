shader Test {
// Note: no closing curly brace
