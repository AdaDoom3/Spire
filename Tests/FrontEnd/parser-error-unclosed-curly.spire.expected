result code = -1
standard error = {
Tests/FrontEnd/parser-error-unclosed-curly.spire(0): error 20001: '}' expected but end of file encountered.
Tests/FrontEnd/parser-error-unclosed-curly.spire(0): error 20001: "inline" expected but end of file encountered.
Tests/FrontEnd/parser-error-unclosed-curly.spire(0): error 20001: "public" expected but end of file encountered.
Tests/FrontEnd/parser-error-unclosed-curly.spire(0): error 20001: "out" expected but end of file encountered.
Tests/FrontEnd/parser-error-unclosed-curly.spire(0): error 20001: "@" expected but end of file encountered.
}
standard output = {
internal compiler error: Operator[]: Index out of Range.
}
